conbench/entities/benchmark_result.py

```diff
@@ -52,24 +54,23 @@ class BenchmarkResult(Base, EntityMixin):
     info_id: Mapped[str] = NotNull(s.String(50), s.ForeignKey("info.id"))
     context_id: Mapped[str] = NotNull(s.String(50), s.ForeignKey("context.id"))
 
-    # Follow
-    # https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#one-to-many
-    # There is a one-to-many relationship between Run (one) and BenchmarkResult
-    # (0, 1, many). Result is child. Run is parent. Child has column with
-    # foreign key, pointing to parent.
-    run_id: Mapped[str] = NotNull(s.Text, s.ForeignKey("run.id"))
-
-    # Between Run (one) and Result (many) there is a one-to-many relationship
-    # that we want to tell SQLAlchemy af ew things about via the following
-    # `relationship()` call. Note that `lazy="select"` is a sane default here.
-    # It means that another SELECT statement is issued upon attribute access
-    # time. `select=immediate` would mean that items are to be loaded from the
-    # DBas the parent is loaded, using a separate SELECT statement _per child_
-    # (which did bite us, in particular for those Run entities associated with
-    # O(1000) Result entities, i.e. hundreds of thousands of SELECT queries
-    # were emitted for building the landing page. A joined query can always be
-    # performed on demand.
-    run: Mapped[Run] = relationship("Run", lazy="select", back_populates="results")
+    # An arbitrary string to group results by CI run. There are no assertions that this
+    # string is non-empty.
+    run_id: Mapped[str] = NotNull(s.Text)
+    # Arbitrary tags for this CI run. Can be empty but not null.
+    run_tags: Mapped[dict] = NotNull(postgresql.JSONB)
```

Comment:
In another thread I asked

> Can we do a flat str/str mapping?

and if we can then we could switch this from type `dict` to something more precise which could help the code base.